senior-frontend-engineer/task.md

```diff
@@ -0,0 +1,29 @@
+# Senior Frontend Engineer
+
+## Description
+Create simple drag and drop layout builder. It should give the ability to build layouts from a set of customizable React components.
+The result of it should be configuration, which can be used by page to render.
+
+## Stack
+Our frontend stack consists of React, ES6, Styled Components and GraphQL.
+
+You are required to use that same stack in this assignment.
+Additional libraries are permitted. 
+
+To speed things up, you can use CRA to bootstrap your app.
+
+## Required functionality:
```

Comment:
It's hard to make this task easier to shorter the time it will take. 

Maybe we could say that we online want to have the order of components that is changed by drag an drop?